packages/paste-website/src/pages/getting-started/design/index.mdx

```diff
@@ -50,39 +50,17 @@ export const pageQuery = graphql`
 
 <content>
 
-## Font files
-
-### Fira Mono
-
-If you are using <strong>any theme</strong> in Paste, you will need
-to [download and install Fira Mono](https://fonts.google.com/specimen/Fira+Mono?query=fira+mono),
-our monospace typeface for code. It’s a free Google font.
-
-### SF Pro
-
-If you are using <strong>any theme</strong> in Paste, you will need to [download and install SF Pro](https://developer.apple.com/fonts/), the
-system font for Apple platforms. We represent native web element styles (like option lists in [Select](/components/select/)) in
-our design assets in the macOS styles.
-
-### Inter
-
-If you are using the <strong>Default theme</strong> in Paste, you will need
-to [download and install Inter](https://github.com/rsms/inter/releases/download/v3.15/Inter-3.15.zip), which we use for UI text.
-It’s a free font. After you download the files, follow the instructions in the
-`install-mac.txt` file in your download. We recommend following the steps using Font Book.
-
-### Whitney
-
-If you are using the <strong>Console theme</strong> in Paste, you will need to download Whitney, which we use for UI text.
-Reach out to us in `#help-design-system` if you need access to the Whitney font files for your work.
+Paste supports Product Designers with tooling and assets that help them build customer UIs. 
```

Comment:
This first line doesn't feel totally necessary - seems a little redundant with the line in the header